### DISCUSSION TITLE
Serial Port Access?

### DISCUSSION THREAD

**Discussion by toml12953** [microsoft/WSL#10780] [Q&A]
WSL1 was great at accessing serial ports. WSL2 is not. I'd like to use the serial ports (some built-in, some on USB - RS-232 converters) in WSL2. Are there any plans to make this facility available? I've tried following the instructions for accessing USB devices but I never get the responses in the instructions.

**Comment by koffes** [comment]
I second this. Proper USB device forward support is a highly requested feature in WSL2.
